features/topology-map/plugins/org.opennms.features.topology.plugins.topo.linkd/src/main/java/org/opennms/features/topology/plugins/topo/linkd/internal/LinkdTopologyProvider.java

```diff
@@ -400,14 +401,41 @@ private void getOspfLinks() {
     }
 
     private void getCdpLinks() {
```

Comment:
I broke the mapping of the links out of this method so we can have and compare 2 implementations: old one and new performance improved one